runelite-client/src/test/java/net/runelite/client/chat/ChatMessageManagerTest.java

```diff
@@ -67,14 +68,14 @@ public void before()
 	}
 
 	@Test
-	public void onChatMessage()
+	public void testMessageRecoloring()
 	{
-		when(chatColorConfig.opaquePublicChat()).thenReturn(Color.decode("#b20000"));
+		when(chatColorConfig.opaqueGameMessage()).thenReturn(Color.decode("#b20000"));
 
 		chatMessageManager.loadColors();
 
 		ChatMessage chatMessage = new ChatMessage();
-		chatMessage.setType(ChatMessageType.PUBLICCHAT);
+		chatMessage.setType(ChatMessageType.CONSOLE);
```

Comment:
Is the chat type for the dodgy neck message really console?

Reply:
whoops, changed to GAMEMESSAGE